admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -90,9 +90,9 @@ systemd
 
 If systemd is installed on the system, a systemd timer could be an alternative to a cronjob.
 
-This approach requires two files: **nextcloudcron.service** and **nextcloudcron.timer**. Create these two files in ``/etc/systemd/system/``.
+This approach requires two files: **nextcloud.service** and **nextcloud.timer**. Create these two files in ``/etc/systemd/system/``.
 
-**nextcloudcron.service** should look like this::
+**nextcloud.service** should look like this::
```

Comment:
I still think that the cron in the `.service` makes sense, because nextcloud itself runs in the apache/php-fpm service. Not that people are confused, because they started the `nextcloud.service` but then the instance is not running at all.

Reply:
Makes sense, @MorrisJobke. I'm also in favor of keeping the `cron` part.